src/composables/state-persistor/state-persistor-rules-write.helper.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  ParamKeys,
+  SP_AVAILABLE_STORAGES,
+  SP_KEY_APPLOGIN,
+  SP_KEY_IPV6,
+  SP_KEY_LOCALFORAGE,
+  SP_KEY_VERSION,
+  StatePersistorRules,
+} from './state-persistor.model'
+
+export default function rulesWriteHelper(): StatePersistorRules & {
```

Comment:
if you have a class, you can maybe share an interface or abstract class with `rulesReadHelper`, seems to be the same signature